css-two-column-layout/styles.css

```diff
@@ -0,0 +1,81 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  font-family: 'Karla', sans-serif;
+}
+
+.container {
+  align-items: center;
+  margin: auto;
+  justify-content: center;
+  width: 900px;
+}
+
+.row {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-between;
+}
+
+.column-full {
+  width: 100%;
+  padding-left: 4rem;
+  padding-right: 4rem;
+}
+
+.column-half {
+  width: 49%;
+  display: flex;
+}
+
+.column-one-third {
+  height: 100%;
+  width: calc(100% / 3);
+}
+
+p {
+  line-height: 1.5;
+}
+
+h1 {
+  text-align: center;
+  font-size: 44px;
+}
+
+h3 {
+  text-align: left;
+}
+
+.column-two-third {
+  height: 100%;
```

Comment:
This shouldn't be necessary. You should only set height on elements when you need them to be a different size from what their content would take up. In this case, we want the content to take as much room as it needs, and no more.